### DISCUSSION TITLE
Sounds in some items seems to be broken

### DISCUSSION THREAD

**Discussion by Alpharius-30-milenium** [FakeFishGames/Barotrauma#13626] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Basically, before update I had the box which is alientrinket1 modded the way that it had music when you equipped it in hands (just for Hellraiser reference) but after update this music started playing always. Removing loop=false and adding targetcontainer did not helped at all.

### Reproduction steps

I think it will be easier if I will just pin archive with xml for it
[bug.zip](https://github.com/Regalis11/Barotrauma/files/13070939/bug.zip)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.1.18.1 (Treacherous Tides)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I unfortunately couldn't find a way to reproduce this, the sound seems to stop playing as soon as I drop the trinket. There doesn't seem to be any logic in the xml for only making the sound play when it's equipped in ones h

https://github.com/Regalis11/Barotrauma/assets/14351070/db8e4d90-e25b-4bad-b939-fabc5e1e74d6

ands, so to me it seems to be working correctly.

**Comment by Alpharius-30-milenium** [comment]
So, I find source of problem. Status effect triggers when item contained in any character, including fractal guardians, so, that part is ok and fixable on my side but I've noticed 1 weird thing: for some reason if I press f3 sound just stops playing sometimes and then triggers again in short period of time. I guess this ticket can be closed as completed

**Comment by Alpharius-30-milenium** [comment]
![image](https://github.com/Regalis11/Barotrauma/assets/60311003/2afe8879-8006-4fad-9958-0cff2871470c)
Well... In multiplayer it plays sound even when it's on the ground
